Week3/homework/Masoud/creditCardValidator/creditCardValidator.js

```diff
@@ -0,0 +1,100 @@
+'use strict'
+
+const validateCreditNumber = (creditCardNumber) => {
+
+  const numStringLength = creditCardNumber.length;
+
+  //Input must be 16 characters
+  function checkLength(creditCardNumber) {
+    if (numStringLength !== 16){
+      return `Invalid! The input ${creditCardNumber} should contains 16 characters!`;
+    } else {
+      return 'success!';
+    }
+  }
+
+  //All characters must be numbers
+  function checkType(creditCardNumber) {
+    const index = [];
+
+    for (let i = 0 ; i < numStringLength ; i++) {
+      if (creditCardNumber[i] == 0 || creditCardNumber[i] == 1 || creditCardNumber[i] == 2 || creditCardNumber[i] == 3 || creditCardNumber[i] == 4 || creditCardNumber[i] == 5 || creditCardNumber[i] == 6 || creditCardNumber[i] == 7 || creditCardNumber[i] == 8 || creditCardNumber[i] == 9) {
```

Comment:
Not a good if statement. Replace this with a regular expression. Check with Amir

Reply:
you can have the function return a boolean and check it when needed instead of  a string